Document new configuration option (#9526)
getsentry/sentry-docs: docs/platforms/javascript/common/user-feedback/configuration/index.mdx

| `messageLabel`       | `Description`                          | The label for the feedback description input field.                |
| `messagePlaceholder` | `What's the bug? What did you expect?` | The placeholder for the feedback description input field.          |
| `successMessageText` | `Thank you for your report!`           | The message to be displayed after a succesful feedback submission. |

> | `isRequiredText`     | `(required)`                           | The text next to a label of a field that needs to be filled.       |

Reviewer: ```suggestion
| `isRequiredText`     | `(required)`                           | The text displayed next to a required field.       |
```
